src/Maker/MakeRegistrationForm.php

```diff
@@ -111,6 +111,10 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
     {
         $interactiveSecurityHelper = new InteractiveSecurityHelper();
 
+        if (null === $this->router) {
+            throw new RuntimeCommandException('Router have been explicitely disabled in your configuration. This command needs to use the router.');
```

Comment:
Help welcome to choose a clearer message.

Reply:
Looks good enough to me - especially for this edge-case